docs/advanced_topics/images/renditions.md

```diff
@@ -38,42 +38,33 @@ See also: [](image_tag)
 
 ## Prefetching image renditions
 
-```{versionadded} 3.0
-This following guidance is only applicable in Wagtail versions 3.0 and above.
+```{versionadded} 4.0
+This following guidance is only applicable in Wagtail versions 4.0 and above.
 ```
 
-When using a queryset to render a list of objects with images, you can make use of Django's built-in `prefetch_related()` queryset method to prefetch the renditions needed for rendering with a single additional query. For long lists of items, or where multiple renditions are used for each item, this can provide a significant boost to performance.
+When using a queryset to render a list of images, you can make use of Wagtail's built-in ``prefetch_renditions`` queryset method to prefetch the renditions needed for rendering with a single additional query. For long lists of items, or where multiple renditions are used for each item, this can provide a significant boost to performance.
 
-For example, say you were rendering a list of events (with thumbnail images for each). Your code might look something like this:
+For example, say you were rendering a list of images. Your code might look something like this:
 
 ```python
-def get_events():
-    return EventPage.objects.live().select_related("listing_image")
+def get_images_uploaded_by_user(user):
+    return Image.objects.filter(uploaded_by_user=user)
 ```
 
 The above can be modified slightly to prefetch the renditions for listing images:
 
 ```python
-def get_events():
-    return EventPage.objects.live().select_related("listing_image").prefetch_related("listing_image__renditions")
+def get_images_uploaded_by_user(user)::
+    return Image.objects.filter(uploaded_by_user=user).prefetch_renditions()
```

Comment:
I don't think this is true, is it?

Reply:
Won't this prefetch no renditions?

Reply:
That's correct. I've changed the `prefetch_renditions` method to prefetch all renditions when no filters are provided.